Automate/Tests/Speckle.Automate.Sdk.Tests.Integration/SpeckleAutomate.cs

```diff
@@ -1,3 +1,4 @@
+using Newtonsoft.Json;
```

Comment:
Same, should this be `Speckle.Newtonsoft.Json`?

Reply:
Resolving, it is correct as is but having both `Speckle.Newtonsoft.Json` and `Newtosoft.Json` in the same project will get weird.

I'll open a ticket for this in the backlog